internal/signer/darwin/signer.go

```diff
@@ -56,6 +56,14 @@ type SignArgs struct {
 	Opts   crypto.SignerOpts // Options for signing, such as Hash identifier.
 }
 
+type EncryptArgs struct {
+	Plaintext []byte
+}
+
+type DecryptArgs struct {
+	Ciphertext []byte
+}
+
```

Comment:
Similar, let's extend this to include the hash type, and default to SHA256 if it is not set